### PR TITLE
home page alignment done and separated html from script

### PR DESCRIPTION
**Pull request by TilakSiriki**

### CODE REVIEW

File: docs/Inflationcalculator/script.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+var slider = document.getElementById('current_cost_slider');
+slider.value = 100000;
+var slider = document.getElementById('rate_of_inflation_slider');
+slider.value = 6.0;
+var slider = document.getElementById('time_period_slider');
+slider.value = 5;
```
Comment (rohinibarla) [inline docs/Inflationcalculator/script.js:6]: Code should not be outside. It must be part of a function.

File: docs/Inflationcalculator/index.html [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <head>
     <title>Inflation calculator</title>
     <link rel="stylesheet" href="../style.css" />
+    <script src="./script.js"></script>
 </head>
 <body onload="updateSliderValue('current_cost_slider', 'current_cost_value'); updateSliderValue('rate_of_inflation_slider', 'rate_of_inflation_value'); updateSliderValue('time_period_slider', 'time_period_value'); calculateResult();">
```
Comment (rohinibarla) [inline docs/Inflationcalculator/index.html:8]: do we really require these function to be called? Check by removing.
Standard is you can call one function. But not these many functions' calls.

File: docs/car_loan_emi/script.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+var slider = document.getElementById('loan_amount_slider');
+slider.value = 1000000;
+var slider = document.getElementById('rate_of_interest_slider');
+slider.value = 6.5;
+var slider = document.getElementById('loan_tenure_slider');
+slider.value = 5;
+        
+function updateSliderValue(slider, valuebox) {
```
Comment (rohinibarla) [inline docs/car_loan_emi/script.js:14]: Find out other way to achieve the same. instead of this way.

File: docs/car_loan_emi/index.html [hunk 4]
```diff
@@ -3,6 +3,7 @@
 <head>
     <title>Car loan emi calculator</title>
     <link rel="stylesheet" href="../style.css" />
+    <script src="./script.js"></script>
 </head>
 <body onload="updateSliderValue('loan_amount_slider', 'principal_amount_value'); updateSliderValue('rate_of_interest_slider', 'rate_of_interest_value'); updateSliderValue('loan_tenure_slider', 'loan_tenure_value'); calculateResult();">
```
Comment (rohinibarla) [inline docs/car_loan_emi/index.html]: Check with https://pagespeed.web.dev/
Before and after this change.

Comment (TilakSiriki) [reply docs/car_loan_emi/index.html]: After changing those things, and verifying with the website, accessibility increased from 80 to 83 and SEO changed from 78 to 89,sir

**Pull request merged**